### ISSUE TITLE
Ending membership does not turn off auto-bill

### ISSUE THREAD

**Issue by mimorita** [nadineproject/nadine#325]
Yesterday I ended member D.G.'s PT5 membership. 
- 7/20 was the last day of the membership
- I set the membership to end on 7/20
- The PT5 membership was on auto-bill with a bill date of the 21st. 

Today D.G. was auto-billed for the PT5. 

Request that changes to membership (particularly ending membership) automatically turn off autobill. 
- I'm pretty sure this was a feature of the old version

**Comment by jsayles** [comment]
Yes it was an important feature and this is a pretty high priority bug.


Jacob

---
Office Nomads - Individuality without Isolation
http://www.officenomads.com -  (206) 323-6500
